draft-ietf-cdni-request-routing-extensions.xml

```diff
@@ -382,7 +405,94 @@
       </section> 
     </section>
 
-      
+
+    <section anchor="cdni-metadata-footprint-types" title="CDNI Metadata Footprint Types">
+      <t>The CDNI Footprint Object is defined in section 4.2.2.2 of the CDNI Metadata interface document
+          <xref target="RFC8006"/>. The object is composed of a footprint-type, referring few of the simple data-types
+          defined in section 4.3 of <xref target="RFC8006"/>, as well as an array of properties from the specified type.
+          Furthermore, in order to facilitate future specification of other optional "footprint-types",
+          a clear process for specifying optional footprint types in an IANA registry is also specified
+          (see section 7.2 of <xref target="RFC8006"/>).
+      </t>
+      <t>Additionally, section 5 in <xref target="RFC8008"/> defines the FCI Capability Advertisement Object,
+          which includes a list of CDNI Footprint Objects as multiple, additive footprint constraints:
+          the advertisement of different footprint types narrows the dCDN's candidacy cumulatively (see Appendix B in
+          <xref target="RFC8008"/>).
+      </t>
+
+      <t>This section specify new data-types to be used also as footprint-types,
+          extending both <xref target="RFC8006"/> and <xref target="RFC8008"/>.
+      </t>
+
+
+      <section anchor="cdni-metadata-ipv4v6cidr-data-type" title="CDNI Metadata IPv4v6CIDR Data Type">
+        <t>Sections 4.3.5 and 4.3.6 of <xref target="RFC8006"/> specify the "IPv4CIDR" and "IPv6CIDR" footprint-types
+           (respectively), for listing different IP addresses blocks. When used as footprint constraints as part of a
+            FCI Advertisement Object, these footprint-types do not allow the combination of IPv4 and IPv6 client addresses.
+            Simply as the constraints are additives and a client address cannot be both IPv4 and IPv6.
```

Comment:
I think what you are missing here is the rational. Why this is a problem? why do you need to be able to define a footprint that uses both ip types? a real life example can be handy.

Reply:
Added. Is that what you meant?